standards/request-response.md

```diff
@@ -868,6 +890,7 @@ The `OPTIONS` method is used to describe communication options for the target re
 - HTTP `OPTIONS` Method **MUST NOT** be used with any sensitive data.
 - HTTP `OPTIONS` Method responses **MUST NOT** be intended to be cached.
 - HTTP `OPTIONS` Method used for `CORS` integration **MUST** provide standard `CORS` headers for access control (`Access-Control-Allow-*`).
+    - Rejected `CORS` requests should return a `403` status code, including the [SPS-CORS-Error](#SPS-CORS-Error) custom header indicating the reason for rejection.
```

Comment:
I can't tell if it's Github or if the indenting is off here

Reply:
Oh wait, it might be indented intentionally :-D

Reply:
yep indentation intended in this case.